### PR TITLE
Allow supplying regulatory domain at runtime

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

File: test/vintage_net/technology/wifi_test.exs [hunk 1]
```diff
@@ -85,6 +85,54 @@ defmodule VintageNet.Technology.WiFiTest do
     assert {:ok, output} == WiFi.to_raw_config("wlan0", input, default_opts())
   end
 
+  test "Set regulatory_domainat runtime" do
```
Comment (fhunleth) [inline test/vintage_net/technology/wifi_test.exs]: `regulatory_domain at` <-- space.

**Comment by ConnorRigby** [comment]
@fhunleth  Yep! just added it

**Comment by fhunleth** [comment]
@ConnorRigby Sorry, I should have been more specific. I was thinking more like around README.md line 224 which is more reference docs rather than example.

**Comment by ConnorRigby** [comment]
@fhunleth sorry about that. Added it there as well

**Pull request merged**